_rules/SC1-2-1-video-only-element-transcript.md

```diff
@@ -0,0 +1,130 @@
+
```

Comment:
This is not rendering as a table. Is it because of this blank line at the start of the file?